components/edit-collective/sections/SetupTwoFactorAuth.js

```diff
@@ -307,10 +434,16 @@ const twoFactorAuthToAccountMutation = gqlV2/* GraphQL */ `
   }
 `;
 
-const addTwoFactorAuthToAccountMutation = graphql(twoFactorAuthToAccountMutation, {
-  name: 'addTwoFactorAuthTokenToIndividual',
-  options: { context: API_V2_CONTEXT },
-});
+const addRemoveTwoFactorAuthFromIndividualMutation = gqlV2/* GraphQL */ `
```

Comment:
That doesn't not make sense to name this variable "addRemoveTwoFactorAuthFromIndividualMutation", it should be `removeTwoFactorAuthFromIndividualMutation`.